### PR TITLE
chore: replace <input /> with <TextField />

### LINKED ISSUE #6742: [CAL-942] Old inputs still being used throughout the app
old input

Search  `<input` to get a rough estimate

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-942](https://linear.app/calcom/issue/CAL-942/old-inputs-still-being-used-throughout-the-app)</sub>

### PR DESCRIPTION
**Pull request by iamsriranga**
## What does this PR do?

replaces the `<input />` tags of type text, number, password with `<TextField />`

Fixes #6742 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: apps/web/components/security/ChangePasswordSection.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { SyntheticEvent, useState } from "react";
 
 import { ErrorCode } from "@calcom/lib/auth";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
-import { Button, showToast } from "@calcom/ui";
+import { Button, showToast, TextField } from "@calcom/ui";
```
Comment (emrysal) [inline apps/web/components/security/ChangePasswordSection.tsx:6]:
```suggestion

// FIXME: This file needs more RHF
```

File: packages/features/ee/workflows/components/AddActionDialog.tsx [hunk 2]
```diff
@@ -202,8 +203,7 @@ export const AddActionDialog = (props: IAddActionDialog) => {
                 </>
               ) : (
                 <div className="mt-5">
-                  <Label>{t("sender_name")}</Label>
-                  <Input type="text" placeholder={SENDER_NAME} {...form.register(`senderName`)} />
+                  <TextField label={t("sender_id")} placeholder={SENDER_ID} {...form.register("senderId")} />
```
Comment (emrysal) [inline packages/features/ee/workflows/components/AddActionDialog.tsx:206]: This is not correct.

**Review by emrysal — CHANGES_REQUESTED** [review]
Code review done, UI review pending 👍 - found one issue, thank you for this!

File: packages/features/ee/workflows/components/AddActionDialog.tsx (hunk 2 above)
Comment (saideepesh000) [reply packages/features/ee/workflows/components/AddActionDialog.tsx:206]: why?, can u describe it more?

Comment (sean-brydon) [reply packages/features/ee/workflows/components/AddActionDialog.tsx:206]:
```suggestion
                  <TextField label={t("sender_id")} placeholder={SENDER_NAME} {...form.register("senderName")} />
```
@emrysal is this what you were referring too?

**Comment by PeerRich** [comment]
@Udit-takkar @G3root can you two test this really extensively? we have a ton of inputs and we need to make sure 100% are working

**Comment by G3root** [comment]
> @Udit-takkar @G3root can you two test this really extensively? we have a ton of inputs and we need to make sure 100% are working

on it 👀

File: packages/features/ee/workflows/components/AddActionDialog.tsx (hunk 2 above)
Comment (saideepesh000) [reply packages/features/ee/workflows/components/AddActionDialog.tsx:206]: Cool, will make the changes

**Comment by PeerRich** [comment]
doubling down again here: very very important that every TextField works. I can already feel that some APIs wont be 100% translateable (whether its visual bugs or data bugs)

File: apps/web/components/booking/pages/AvailabilityPage.tsx [hunk 3]
```diff
@@ -179,9 +179,9 @@ const AvailabilityPage = ({ profile, eventType, ...restProps }: Props) => {
                             <p className="mb-1 -ml-2 inline px-2 py-1">
                               {getRecurringFreq({ t, recurringEvent: eventType.recurringEvent })}
                             </p>
-                            <input
+                            <TextField
                               type="number"
-                              min="1"
```
Comment (hariombalhara) [inline apps/web/components/booking/pages/AvailabilityPage.tsx:184]: This breaks the UI.
After:
image
Before:
image

**Review by hariombalhara — CHANGES_REQUESTED** [review]
@saideepesh000 Can you please test all the changes visually as well.
As @PeerRich mentioned, all cases might not be simply mappable. 

Also, while testing please keep in mind that Booking Pages have Dark Mode as well(and inputs should be tested in that also). Plus, do test your changes in one of the smaller screens

File: apps/web/components/booking/pages/AvailabilityPage.tsx (hunk 3 above)
Comment (saideepesh000) [reply apps/web/components/booking/pages/AvailabilityPage.tsx:184]: sure

**Comment by PeerRich** [comment]
i think we need to refactor input to TextField one by one, one PR for each page and thoroughly test it

closing for now, @sydwardrae can send you a gift for your work!

**Comment by saideepesh000** [comment]
yeah, agreeded @PeerRich

**Pull request closed without merging**